src/main/java/core/basesyntax/IsoscelesTrapezoidSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoidSupplier {
+    private static final RandomGeometricValue randomSize = new RandomGeometricValue();
+    private static final ColorSupplier color = new ColorSupplier();
+
+    public Figure getRandomIsoscelesTrapezoid() {
+        return new IsoscelesTrapezoid(randomSize.getRandomGeometricValue(),

```

Comment:
The IsoscelesTrapezoid constructor is called with three random values. An IsoscelesTrapezoid usually has two equal sides (bases) and height, so you should adjust the parameters accordingly.